CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.1
+* Fix a bug where trips were sorted by trip id instead of by time they
```

Comment:
How about
"Fixes a bug where trips were sorted by trip_id rather than vehicle arrival time"

Reply:
oh, i thought it was preferable to be present progressive or whatever the term is for the tense i used.  i can go your way since it makes more sense englishwise

Reply:
i accidentally amended my previous commit so there isnt a new commit but its updated.